src/notebooks/R Mia Examples/Fetch-Analyses-metadata-for-a-Study.qmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "Fetch Analyses metadata for a Study"
+author:
+  - name: Noah de Gunst
+    affiliation:
+      - id: mia
+        name: miaverse team
+categories: [R]
+execute: 
+  eval: true
+---
+
+::: {style="max-width:1200px"}
+![](./../_resources/mgnify_logo.png)
+
+# Fetch a Study using MGnifyR; download the metadata for all of its Analyses
+
+The [MGnify API](https://www.ebi.ac.uk/metagenomics/api/v1) returns data and relationships as JSON. [MGnifyR](https://github.com/beadyallen/MGnifyR) is a package to help you read MGnify data into your R analyses.
```

Comment:
https://github.com/beadyallen/MGnifyR is old address. Now the repo is under EBI. Moreover, I would link to this https://www.bioconductor.org/packages/release/bioc/html/MGnifyR.html

Reply:
This was not taken into account

Reply:
sorry done now